deploy/cache-writer.yaml

```diff
@@ -18,7 +18,166 @@ kind: Template
 metadata:
   name: ccx-cache-writer
 objects:
-# TODO: Add the cache-writer deployment + services
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: valkey-writer
+    annotations:
+      ignore-check.kube-linter.io/minimum-three-replicas: "This app doesn't have that much traffic"
+  spec:
+    inMemoryDb: true
+    envName: ${ENV_NAME}
+    testing:
+      iqePlugin: ccx
+    deployments:
+      - name: db-writer
+        minReplicas: ${{CACHE_WRITER_REPLICAS}}
```

Comment:
I think minReplicas is deprecated in clowder spec

Reply:
Replaced with `replicas`